docs/src/czt.md

```diff
@@ -1,11 +1,14 @@
 # CZTs
 Chirp Z Transformations: Allows Fourier-transformation and at the same time zooming into the result,
 which is why it is also called the Zoomed-FFT algorithm.
-The algorithm is loosely based on a publication [Rabiner, Schafer, Rader, The Cirp z-Transform Algorithm, IEEE Trans AU 17(1969) p. 86] and a 2D Matlab Version written by N.G. Worku & H. Gross, with their consent (28. Oct. 2020) to make it openly available. It currently needs three FFTs to perform its work.
-As one of these FFTs only depends on the datasize and zoom parameters, it can be moved to a plan in future implementations.
-
+The algorithm is loosely based on a publication [Rabiner, Schafer, Rader, The Chirp z-Transform Algorithm, IEEE Trans AU 17(1969) p. 86]. It needs three FFTs to perform its work but one can be precalculated by using `plan_czt`.
+Variable zooms, transform dimensions, array center positions as well as output sizes are supported along wiht a low-level interface by specifingy `a` and `w`. 
 ```@docs
 FourierTools.czt
+FourierTools.plan_czt
 FourierTools.iczt
 FourierTools.czt_1d
+FourierTools.plan_czt_1d
+FourierTools.CZTPlan_1D
+FourierTools.CZTPlan_ND
```

Comment:
Why exporting the custom types?

Reply:
I agree. They should remain local.
Fixed.